src/workerd/api/worker-loader.h

```diff
@@ -59,8 +59,9 @@ class WorkerLoader: public jsg::Object {
     jsg::Optional<kj::Array<const byte>> data;  // byte blob, imports as ArrayBuffer
     jsg::Optional<jsg::Value> json;             // arbitrary JS value, will be serialized to JSON
                                                 // and then parsed again when imported
+    jsg::Optional<kj::String> py;               // Python module
```

Comment:
What about `pythonRequirement`? (I don't actually know what it does, but notice it's not represented here...)

Reply:
It's used for runtime-provided python packages.
We're moving away from it short term as it creates odd conflicts and provides little actual value outside of some small memory sharing opportunity.
We intend to revisit it later after we're done with UX and performance.

All that to say, because we are no longer suggesting it for regular python workers, I don't think we need to support it for dynamic python worker loading at the moment.

Using the vendoring approach is better for the users at the moment.